news/2022-04-24-beatmap-spotlights-season-5-winter-2022-conclusion.md

```diff
@@ -0,0 +1,260 @@
+---
+layout: post
+title: "Beatmap Spotlights Winter 2022 Conclusion & Players' Pick Voting"
+date: 2022-04-24 16:00:00 +0000
+---
+
+The Winter 2022 season of the Beatmap Spotlights has come to an end - come check out the top 50 for each gamemode, and vote on your favourite beatmaps from the season!
+
+![](/wiki/shared/news/banners/beatmap-spotlights.jpg)
+
+By far the most played Beatmap Spotlights yet, over **3300** players have battled in the Winter 2022 playlists for fun and glory. To keep things fresh, we've also decided to try something new that we've titled as the Players' Pick. Read on below for more details.
+
+## Beatmap of the Season - Players' Pick
+
+Previously, we've held a *Beatmap of the Season* competition where the [Spotlights Curators](/wiki/Beatmap_Spotlights#curators) would vote for their favourite entries from season's beatmaps pool. This time however, we'd like to expand further and let the players select their most-loved beatmaps too!
+
+The results of both the curator and player pick will be announced at the the launch of the Spring 2022 season of the Beatmap Spotlights, and the creators of winning entries will be awarded with **three months of osu!supporter tag** each. If you'd like to make your opinions known on the best maps of the season, this is the place to do it!
+
+Check out each respective game mode's contest and vote today:
+
+- [osu!](https://osu.ppy.sh/community/contests/143)
+- [osu!taiko](https://osu.ppy.sh/community/contests/144)
+- [osu!catch](https://osu.ppy.sh/community/contests/145)
+- [osu!mania](https://osu.ppy.sh/community/contests/146)
+
+The voting will remain open for **7 days** from the date of the post, so be sure to vote in time!
+
+## Final rankings
+
+### osu!
+
+| Placement | User | Tier |
+| --: | :-- | :-: |
+| #1 | [fieryrage](https://osu.ppy.sh/users/3533958) | ![](https://assets.ppy.sh/profile-badges/spotlights-2022/winter-2022-osu!-ri-3.png "Rhythm Incarnate 2") |
```

Comment:
hold on we do have these badges in the wiki [right here](https://github.com/ppy/osu-wiki/tree/master/wiki/Beatmap_Spotlights/img/badges/winter_2022/osu)

also can these be shortened to maybe top 10? they make the post way too long imo and there's already a spreadsheet. ideally we could put these ranking tables side by side to maximise space usage (at which point maybe top 50 would be feasible), but with this the tables just get needlessly long and narrow

Reply:
hmm, these are osu! only, not sure if it's worth to create such kinda inconsistency here?

Reply:
>also can these be shortened to maybe top 10?

Not sure about top 10 only, because at least for osu! you wouldn't even be able to see all the Rhythm Incarnate Players. I think those should at least be shown.

Reply:
> hmm, these are osu! only, not sure if it's worth to create such kinda inconsistency here?

well fair enough, sort of, but the badges would still be best put in the wiki just like the the [summer 2020 badges](https://github.com/ppy/osu-wiki/tree/master/wiki/Beatmap_Spotlights/img/badges/summer_2020), and i'm not sure why the badges for the other modes weren't in the beatmap spotlights article in the first place

Reply:
> Not sure about top 10 only, because at least for osu! you wouldn't even be able to see all the Rhythm Incarnate Players. I think those should at least be shown.

idea:

![image](https://user-images.githubusercontent.com/36758269/164991745-15ccb39c-3e25-4a37-9652-9d1b7838ec1b.png)

@Ephemeralis what would you think? maybe there's a better solution

Reply:
above pic looks good